Components/input.css

```diff
@@ -0,0 +1,36 @@
+.input-container{
+    padding: 1rem;
+    border: 1px solid var(--color-gray-200);
+    width: 60%;
+}
+.input-form{
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    justify-content: space-evenly;
+    background-color: var(--color-gray-100);
+    padding: 0.5rem;
+    border: 1px solid var(--color-gray-600);
+    box-shadow: 5px 5px 5px var(--color-gray-300);
+}
+.input-password-text{
+    color: var(--color-error);
+    margin: 0.3rem;
+}
+.input-form > *{
+    margin:0.3rem;
+}
+#password{
+    border: 2px var(--color-error) solid;
+}
+.input-form input,#description {
+    width: 80%;
+}
+#form-submit{
+    width: 40%;
```

Comment:
avoid using id selectors in css https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4, refer this article

Reply:
great read..... done replacing id with class.